packages/api/src/command/medical/patient/append-doc-query-progress.ts

```diff
@@ -36,6 +46,7 @@ export async function appendDocQueryProgress({
   increaseCountConvertible,
   reset,
```

Comment:
Similar to the point i made [here](https://github.com/metriport/metriport/pull/1363#discussion_r1434307624) when we do something like reset is it per HIE are all of them?